client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-expanded-content/boost-site-performance.tsx

```diff
@@ -112,15 +113,29 @@ export default function BoostSitePerformance( {
 					</div>
 				</div>
 				<div className="site-expanded-content__card-footer">
-					<Button
-						href={ href }
-						target="_blank"
-						onClick={ () => trackEvent( 'expandable_block_optimize_css_click' ) }
-						className="site-expanded-content__card-button"
-						compact
-					>
-						{ translate( 'Optimize CSS' ) }
-					</Button>
+					{ hasBoost && (
+						<Button
+							href={ OptimizeCSSUrl }
+							target="_blank"
+							onClick={ () => trackEvent( 'expandable_block_optimize_css_click' ) }
+							className="site-expanded-content__card-button"
+							compact
+						>
+							{ translate( 'Optimize CSS' ) }
+						</Button>
+					) }
+
+					{ ! hasBoost && (
+						<Button
+							href={ ConfigureBoostUrl }
+							target="_blank"
+							onClick={ () => trackEvent( 'expandable_block_configure_boost_click' ) }
+							className="site-expanded-content__card-button"
+							compact
+						>
+							{ translate( 'Configure Boost' ) }
+						</Button>
+					) }
```

Comment:
I can we can write this as below. WDYT?

```
const buttonProps = hasBoost
  ? {
      label: 'Optimize CSS',
      href: OptimizeCSSUrl,
      onClick: () => trackEvent('expandable_block_optimize_css_click'),
    }
  : {
      label: 'Configure Boost',
      href: ConfigureBoostUrl,
      onClick: () => trackEvent('expandable_block_configure_boost_click'),
    };

return (
  <Button
    href={buttonProps.href}
    target="_blank"
    onClick={buttonProps.onClick}
    className="site-expanded-content__card-button"
    compact
  >
    {translate(buttonProps.label)}
  </Button>
);
```

Reply:
Good suggestion. I have applied this to my changes. thanks 👍